src/shipit_static_analysis/shipit_static_analysis/revisions.py

```diff
@@ -92,17 +87,6 @@ def __init__(self, description, api):
         self.phid = diff['fields']['revisionPHID']
```

Comment:
Is this used anywhere now? If it isn't, it could be a local variable instead of a property

Reply:
Or remove it altogether, after all the assertion is probably not required?